src/main/java/com/alipay/oceanbase/rpc/table/ObTable.java

```diff
@@ -394,8 +391,28 @@ public ObPayload execute(final ObPayload request) throws RemotingException,
         } catch (Exception ex) {
             throw new ObTableConnectionStatusException("check status failed", ex);
         }
-
-        return realClient.invokeSync(connection, request, obTableExecuteTimeout);
+        boolean needReconnect = false;
```

Comment:
Could we extract a function called ```executeWithReconnect``` or ```invokeSyncWithReconnect```?

Reply:
done